src/main/java/com/minecolonies/entity/ai/citizen/fisherman/EntityAIWorkFisherman.java

```diff
@@ -212,6 +202,15 @@ private AIState prepareForFishing()
     {
         if (checkOrRequestItems(new ItemStack(Items.FISHING_ROD)))
         {
+            if(worker != null && worker.isFemale())
+            {
+                SoundUtils.playSoundAtCitizenWithChance(world, worker.getPosition(), FishermanSounds.Female.needFishingRod, CHANCE_TO_PLAY_SOUND);
+            }
+            else
+            {
+                SoundUtils.playSoundAtCitizenWithChance(world, worker.getPosition(), FishermanSounds.Male.needFishingRod, CHANCE_TO_PLAY_SOUND);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'worker' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2259)